pkg/koordlet/metricsadvisor/collector.go

```diff
@@ -158,6 +158,10 @@ func (c *collector) Run(stopCh <-chan struct{}) error {
 			return
 		}
 		ic.collectContainerCPI()
+		// todo: split cpi collector and psi collector into independent feature gates
+		ic.collectContainerPSI()
```

Comment:
reduce the collect interval of PSI to 10s

Reply:
Also split cpi & psi collectors apart